packages/medusa/src/api/routes/admin/product-tags/index.ts

```diff
@@ -31,12 +31,26 @@ export const defaultAdminProductTagsFields = [
 ]
 export const defaultAdminProductTagsRelations = []
 
+/**
+ * @schema AdminProductTagsListRes
+ * type: object
+ * properties:
+ *   product_tags:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/ProductTag"
+ *   count:
+ *     type: integer
+ *     description: The total number of items available
+ *   offset:
+ *     type: integer
+ *     description: The number of items skipped before these items
+ *   limit:
+ *     type: integer
+ *     description: The number of items per page
```

Comment:
Forgive my OAS newbie-ness, but would something like this work to make things easier for these repetitive cases?
```
allOf:
  - $ref: '#/components/schemas/ListPaginationProperties'
  - type: object
    properties:
      product_types:
        type: array
        items:
          $ref: "#/components/schemas/ProductType"
```

Reply:
Yes, I thought about the same optimization while I was working on this. I believe it would work. I'll create a task in Linear to look into it in a future effort. CORE-1004